MapboxNavigation/NavigationViewControllerDelegate.swift

```diff
@@ -191,5 +175,16 @@ public protocol NavigationViewControllerDelegate: VisualInstructionDelegate {
      */
     @objc(navigationViewController:roadNameAtLocation:)
     optional func navigationViewController(_ navigationViewController: NavigationViewController, roadNameAt location: CLLocation) -> String?
+    
+    
+    //MARK: Obsolete
+    
+    @available(*, obsoleted: 0.1, message: "The NavigationViewController no longer forwards MGLMapView messages. Use MGLMapViewDelegate.mapView(_:imageFor:) instead.")
```

Comment:
Technically these are MGLMapViewDelegate messages, but anyways I think we should keep these messages as brief as possible, because Xcode only shows a line or two of the message by default (including the standard deprecation warning message), and it shows even less in the source code editor’s margin.

```suggestion
    @available(*, obsoleted: 0.1, message: "Use MGLMapViewDelegate.mapView(_:imageFor:) instead.")
```

(×2)